packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PickerItemsDefault.scss

```diff
@@ -28,5 +28,10 @@
     vertical-align: middle;
   }
 
-
+  // People Picker specific override to prevent truncation of Persona name
+  .ms-Persona {
```

Comment:
This doesn't make sense. The comment says "People Picker", but the class is .ms-Persona, which is redundant.